src/webhooks/resources/verify_header.js

```diff
@@ -0,0 +1,104 @@
+const crypto = require('crypto');
+const DEFAULT_TOLERANCE = 300; // 5 minutes
+const EXPECTED_SCHEME = 'v1';
+
+const Base = require('../../base');
+
+/**
+ * Secure compare, from https://github.com/freewil/scmp
+*/
+function _secureCompare (a, b) {
```

Comment:
move to `utils/` ?

Reply:
Moving this to utils would make sense to me.